src/GovUk.Education.ExploreEducationStatistics.Data.Seed/SamplePublications.cs

```diff
@@ -34,29 +34,24 @@ public static class SamplePublications
                 {22, DataCsvFile.SEN2_MI},
                 {23, DataCsvFile.skeleton_dashboard_tidy_data_NARTS},
                 {24, DataCsvFile.skeleton_dashboard_tidy_data_annual_v4},
-                {25, DataCsvFile.clean_data_fe},
-                {26, DataCsvFile.level_2_3_national},
-                {27, DataCsvFile.level_2_3_sf},
-                {28, DataCsvFile.level_2_3_sfla},
-                {29, DataCsvFile.KS2_2016_test_UD},
-                {30, DataCsvFile.KS4_2018_LA_Char_Testdata},
-                {31, DataCsvFile.KS4_2018_Nat_Char_Testdata},
-                {32, DataCsvFile.KS4_2018_Subject_Tables_S1_TestData},
-                {33, DataCsvFile.KS4_2018_Subject_Tables_S3_TestData}
+                
+                {25, DataCsvFile.level_2_3_national},
+                {26, DataCsvFile.level_2_3_sf},
+                {27, DataCsvFile.level_2_3_sfla},
+                {28, DataCsvFile.KS2_2016_test_UD},
+                {29, DataCsvFile.KS4_2018_LA_Char_Testdata},
+                {30, DataCsvFile.KS4_2018_Nat_Char_Testdata},
+                {31, DataCsvFile.KS4_2018_Subject_Tables_S1_TestData},
+                {32, DataCsvFile.KS4_2018_Subject_Tables_S3_TestData},
+                
+                {33, DataCsvFile.clean_data_fe}
```

Comment:
Should this be in here?

Reply:
Yeah - clean_data is the only ignored file now.

Reply:
done